### PR TITLE
Code updates dec9 dec11 2019

### PR DESCRIPTION
**Pull request by pegahvaezi**

### CODE REVIEW

File: src/components/VerticalActionsList.js [hunk 1]
```diff
@@ -96,6 +105,21 @@ function VerticalActionsList({
       {viewsOpen.priority && (
         <Priority onClose={() => setViews({ ...defaultViews })} />
       )}
+      {viewsOpen.archive && (
+        <AlertPopupTemplate
+          onClose={() => setViews({ ...defaultViews })}
+          className='archive_popup'
+          heading='Archiving'
+          // TODO : make the archiving message dynamic, showing the title(s) of the card(s) selected for archiving //
+          content={archiveMessagePartOne, archiveMessagePartTwo}
```
Comment (Connoropolous) [inline src/components/VerticalActionsList.js]: Did you verify that this `archiveMessagePartOne, archiveMessagePartTwo` actually works to display the intended message? By the looks of it to me the syntax wouldn't work.

File: src/components/HierarchyPicker/HierarchyPicker.js [hunk 2]
```diff
@@ -66,27 +71,39 @@ HierarchyPicker.propTypes = {
...
+          />
         )}
-      </div>
-      <span>
         {selected && (
-          <img src='img/check-mark-line.svg' alt='check' width='50%' />
+          <Icon
+            name='radio_button_checked.svg'
+            size='small'
+            className='radio_button'
```
Comment (Connoropolous) [inline src/components/HierarchyPicker/HierarchyPicker.js]:
```suggestion
```

Duplicate `className` declaration

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -150,7 +151,7 @@
   z-index: -1;
 }
 
-.guidebook_open_help .icon {
```
Comment (Connoropolous) [inline src/components/Header/Header.css:153]: why replace .icon with img ... I don't think that will work?

**Review by Connoropolous — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.css (hunk 3 above)
Comment (pegahvaezi) [reply src/components/Header/Header.css:153]: this one isn't an icon

File: src/components/VerticalActionsList.js (hunk 1 above)
Comment (pegahvaezi) [reply src/components/VerticalActionsList.js]: fixed

**Review by Connoropolous — APPROVED** [review]

**Pull request merged**